addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -773,15 +772,14 @@ def unpickable_item(self,
         the backorder (by default it is canceled).
```

Comment:
By default is not cancelled anymore, can you remove this although it is not related to your changes, please?